ArduCopter/Parameters.cpp

```diff
@@ -173,7 +173,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: FS_GCS_ENABLE
     // @DisplayName: Ground Station Failsafe Enable
     // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. NB. The GCS Failsafe is only active when RC_OVERRIDE is being used to control the vehicle.
-    // @Values: 0:Disabled,1:Enabled always RTL,2:Enabled Continue with Mission in Auto Mode,3:Enabled always SmartRTL or RTL,4:Enabled always SmartRTL or Land
+    // @Values: 0:Disabled,1:Enabled always RTL,2:Deprecated,3:Enabled always SmartRTL or RTL,4:Enabled always SmartRTL or Land,5:Enabled always land
```

Comment:
This will change what users see when running 3.6 (and earlier) so we should add a comment like "Deprecated in 4.0".

Reply:
Done.